backend/server/services/game.service.ts

```diff
@@ -159,14 +161,14 @@ export default class GameService {
       'There are open rounds.'
     );
 
-    const newGameState = _.cloneDeep(gameState);
-    const activePlayers = newGameState.players.filter((p) => p.isActive);
+    let newGameState = _.cloneDeep(gameState);
```

Comment:
We could also just import the functions we need, instead of all of lodash (Meaning `import { cloneDeep, last } from 'lodash'`). Wydt?

Reply:
Does not really make a difference for the backend (might increase the boot time, but not significantly), on the frontend we do do that to minimize the package size